src/main/java/org/minions/devfund/mauricioz/MovieDatabase.java

```diff
@@ -0,0 +1,158 @@
+package org.minions.devfund.mauricioz;
+
+import java.util.ArrayList;
+
+/**
+ * move data base class definition.
+ */
+public class MovieDatabase {
+    private ArrayList<Movie> movies;
+    private ArrayList<Actor> actors;
+
+    /**
+     * constructor.
+     */
+    public MovieDatabase() {
+        this.movies = new ArrayList<Movie>();
+        this.actors = new ArrayList<Actor>();
+    }
+
+    /**
+     * add a new movie in the db.
+     * @param name movie name.
+     * @param actors casting list.
+     */
+    public void addMovie(final String name, final String[] actors) {
+        if (!isAlreadyRegistered(name)) {
+            Movie newMovie = new Movie(name);
+            movies.add(newMovie);
+            Actor actor;
+            ArrayList<Actor> casting = new ArrayList<Actor>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)